### ISSUE TITLE
Model.update() not firing pre_save signal

### ISSUE THREAD

**Issue by rvindas-igd** [coleifer/peewee#703]
I have this import in my models: `from playhouse.signals import Model, pre_save` and have a model that inherits from that 'Model' class. I also added the pre_save signal implementation which gets fired when I execute Model.create() or Model.save().
But executing a Model.update().execute() updates the record correctly but it doesn't fire the pre_save signal.

**Comment by coleifer** [comment]
Yes this is a known (and obvious) limitation. You'll want to look into triggers.

**Comment by jimjkelly** [comment]
Not looking to resurrect the dead here, but could there perhaps be a docs update to reflect this? It says:

> Provides an additional keyword argument created, indicating whether the model is being saved for the first time or updated.

which to me made me expect it would work, and thus chase my tail until I came here.  :)

Thanks!

**Comment by coleifer** [comment]
See 590ccac3044e13988270017b85ec85dd76161c8b

**Comment by jimjkelly** [comment]
Thanks!
